applications/asset_tracker_v2/overlay-lwm2m.conf

```diff
@@ -0,0 +1,78 @@
+CONFIG_LWM2M_INTEGRATION=y
+CONFIG_LWM2M=y
+CONFIG_LWM2M_DTLS_SUPPORT=y
+CONFIG_LWM2M_SECURITY_KEY_SIZE=33
+CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_ENGINE_STACK_SIZE=3072
+CONFIG_LWM2M_COAP_MAX_MSG_SIZE=2304
+CONFIG_LWM2M_COAP_BLOCK_SIZE=1024
+CONFIG_LWM2M_DNS_SUPPORT=y
+CONFIG_LWM2M_RW_JSON_SUPPORT=n
+CONFIG_LWM2M_SECURITY_INSTANCE_COUNT=3
+CONFIG_COAP_INIT_ACK_TIMEOUT_MS=4000
+CONFIG_BASE64=y
+
+# Enable TLS session caching to prevent doing a full TLS handshake for every send.
+CONFIG_LWM2M_TLS_SESSION_CACHING=y
+
+# Set uptime to once every 12 hours.
+CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME=43200
```

Comment:
Maybe for longer lifetime LWM2M_SECONDS_TO_UPDATE_EARLY this should be added and config some minutes before timeout

Reply:
I was a little bit unsure how to set `LWM2M_SECONDS_TO_UPDATE_EARLY` properly. I wonder, what happens if the server decides to set the lifetime to something lower than `LWM2M_SECONDS_TO_UPDATE_EARLY`?

We could do one of two things. Increase `LWM2M_SECONDS_TO_UPDATE_EARLY` to something more sane (like 5-10 minutes). This way we know that we've had an honest try to send an UPDATE before the timer runs out, if the application sends data rarer than 12 hours and we are in poor network conditions. Or, we could leave it as is and document that if the application sends data less than every 12 hours `LWM2M_SECONDS_TO_UPDATE_EARLY` should probably be increased. Especially in poor network conditions.

Reply:
Ok. Lets think this after 2.0. Engine need to be smarter. Because if we set 5min and user set from Coiote timeout shorter it may affect aggressive Update loop.